backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/utils/CpuPinningHelper.java

```diff
@@ -100,15 +102,55 @@ private static List<Integer> createRange(int start, int end) {
     }
 
     /**
-     * Provides a CPU pinning map based on the VdsCpuUnit list. The pinning will be based on the list order.
+     * Provides a dedicated CPU pinning map based on the VdsCpuUnit list. The pinning will be based on the list order.
```

Comment:
Exclusive cpu pinning...

Reply:
Here it's actually specific for dedicated.